v2.6/getting-started/kubernetes/installation/hosted/canal/index.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Canal/flannel Hosted Install
+canonical_url: 'https://docs.projectcalico.org/v2.6/getting-started/kubernetes/installation/hosted/canal/'
```

Comment:
This should be v3.0

Reply:
I don't think so - we don't have a v3.0 version of canal yet.

Reply:
Although I don't see the v3.0 copy in this PR, hm

Reply:
Actually, you can leave it at v2.6 since it's not in v3.0

Reply:
I assume that is only if this PR also added the v3.0 page.